test/sql/test_functions.py

```diff
@@ -219,6 +230,34 @@ class myfunc(GenericFunction):
 
         assert isinstance(func.myfunc().type, DateTime)
 
+    def test_case_sensitive(self):
+        class MYFUNC(GenericFunction):
+            type = DateTime
+
+        assert isinstance(func.MYFUNC().type, DateTime)
+        assert isinstance(func.MyFunc().type, DateTime)
+        assert isinstance(func.mYfUnC().type, DateTime)
+        assert isinstance(func.myfunc().type, DateTime)
+
+    def test_replace_function(self):
+
+        class replacable_func(GenericFunction):
+            type = Integer
+            identifier = 'replacable_func'
+
+        assert isinstance(func.Replacable_Func().type, Integer)
+        assert isinstance(func.RePlAcaBlE_fUnC().type, Integer)
+        assert isinstance(func.replacable_func().type, Integer)
+
+        with expect_warnings():
```

Comment:
**mike bayer** (zzzeek) wrote:

the expect_warnings() here should have at least part of the text of the warning that's being emitted to match it up.   At the moment the expect_warnings() context manager is otherwise not asserting that a warning was definitely emitted (this is a bug in the test fixture).